<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleWindow</class>
 <widget class="QMainWindow" name="TitleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="player1label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="player1score">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="gametitle">
      <property name="styleSheet">
       <string notr="true">font: 32pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Tic-Tac-Toe</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QLabel" name="imagelabel">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>images/icon.ico</pixmap>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="newgameButton">
      <property name="text">
       <string>New</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="registerbutton">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="TicTacToeWidget" name="tictactoe" native="true"/>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="player2label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="player2score">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 12pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewgame"/>
    <addaction name="actionQuitgame"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewgame"/>
   <addaction name="actionQuitgame"/>
  </widget>
  <action name="actionNewgame">
   <property name="icon">
    <iconset>
     <normalon>:/images/icons8-new-48.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Newgame</string>
   </property>
   <property name="toolTip">
    <string>New Game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionQuitgame">
   <property name="icon">
    <iconset>
     <normalon>:/images/icons8-close-window-48.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>QuitGame</string>
   </property>
   <property name="toolTip">
    <string>Quit Game</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TicTacToeWidget</class>
   <extends>QWidget</extends>
   <header>tictactoewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
